### PR TITLE
telegram-desktop-bin: bump version to 1.2.15

### PR DESCRIPTION
**Pull request by jvesely**
Signed-off-by: Jan Vesely <jano.vesely@gmail.com>

### CODE REVIEW

File: net-im/telegram-desktop-bin/telegram-desktop-bin-1.2.14.ebuild [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils gnome2-utils xdg
```
Comment (Zlogene) [inline net-im/telegram-desktop-bin/telegram-desktop-bin-1.2.14.ebuild]: please use `desktop` eclass rather than `eutils` one

**Comment by henning-schild** [comment]
LGTM, please merge

**Comment by henning-schild** [comment]
@jvesely if you want also include a commit that drops 1.2.1, otherwise i will do that on top of this one

**Comment by NP-Hardass** [comment]
@henning-schild @jvesely I'll merge within an hour or two when I have a chance.

**Comment by henning-schild** [comment]
That PR took long enough for upstream to come with the next release. I would suggest to drop 1.2.14 and jump right to 1.2.15 with the drop on top. Or keep both new ones and drop 1.2.6 on top of 15.

**Comment by NP-Hardass** [comment]
@henning-schild Not sure I follow your logic on dropping old versions.  What are you attempting to maintain? Newest only? Newest 2?  Newest in repo + newest in PR?

**Comment by henning-schild** [comment]
@NP-Hardass i am not sure which versions to keep in the tree, two recent ones seems good

I do not know what bugs where fixed in 1.2.14 but since 15 came fast i can only assume that 14 might not be worth keeping. Let us just keep the three we have until the next release.

**Pull request closed without merging**